workers/src/abi.rs

```diff
@@ -0,0 +1,41 @@
+//! Types create from the JSON ABI files.
+//!
+//! For example, to be able to decode the logs' data, or call contracts' methods.
+//!
+//! To obtain the corresponding ABI, there are two ways:
+//!   - Manually downloading the ABI from the contract's source code (we use this one for now);
```

Comment:
I just re-reviewed this part. I'd encourage us to have these contracts in codebase at least for now.
Project which you `git clone` and doesn't compile without some additional manual tweaking smells.